tests/e2e/performance/csi_tests/test_pvc_bulk_clone_performance.py

```diff
@@ -257,58 +282,29 @@ def test_bulk_clone_performance(self, namespace, tmp_path):
             # Create text file with results of all subtest (3 - according to the parameters)
             self.write_result_to_file(res_link)
 
-        # Finally is used to clean-up the resources created
+        # Finally, is used to clean up the resources created
         # Irrespective of try block pass/fail finally will be executed.
         finally:
             # Cleanup activities
             log.info("Cleanup of all the resources created during test execution")
-            if job_pod_file:
-                job_pod_file.delete(namespace=self.namespace)
-                job_pod_file.wait_for_delete(
-                    resource_name=job_pod_file.name, namespace=self.namespace
-                )
-
-            if job_clone_file:
-                job_clone_file.delete(namespace=self.namespace)
-                job_clone_file.wait_for_delete(
-                    resource_name=job_clone_file.name, namespace=self.namespace
-                )
-
-            if job_pvc_file:
-                job_pvc_file.delete(namespace=self.namespace)
-                job_pvc_file.wait_for_delete(
-                    resource_name=job_pvc_file.name, namespace=self.namespace
-                )
+            for object_file in [job_pod_file, job_clone_file, job_pvc_file]:
+                if object_file:
+                    object_file.delete(namespace=self.namespace)
+                    object_file.wait_for_delete(
+                        resource_name=object_file.name, namespace=self.namespace
```

Comment:
@Avilir Can we add validation of data cleanup too?

Reply:
Done.